### DISCUSSION TITLE
LocalTV+

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#568] [Q&A]
Are the channels from LocalTV+ going to be added?
https://www.pokharel.org/

**Comment by BellezaEmporium** [comment]
If there's any link with LocalNOW, no. If it's purely independent, you'll need to ask the provider if they're alright into getting those playlists in here.

**Reply by hackwin** [reply]
Any interest in covering the NYC area?  I have a collection of hdhomerun quatro and live about 8 miles away.

**Reply by BellezaEmporium** [reply]
The interest is there everywhere. We accept a bit of everything, as long as it's not Xtream-Codes.

**Reply by okc** [reply]
Local Tv + has nothing to do with Byron Allen’s Local Now.  Developer has released polished apps to TestFlight for iOS , iPadOS, and Apple TV, but no source code.  Live, quality primary feeds of local pbs, cbs, abc, nbc, fox, cw affiliates and WHDH, no other subchannels.  No details about the underlying streams

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
